### PR TITLE
Split configure postoffice command

### PR DESCRIPTION
**Pull request by jjponz**
- Split command configure post office in two
    - configure_post_office_publishers
    - configure_post_office_topics

- Fix some linting errors

### CODE REVIEW

File: postoffice_django/config.py [hunk 1]
```diff
@@ -32,12 +42,24 @@ def _create_publishers(consumer):
...
+        )
+    except (ConnectTimeout, ConnectionError):
+        return False
 
-def _save_creation_result_log(response):
-    if response.status_code == 201:
-        logger.info(f'{response.url} succesfully creation')
-    else:
-        logger.error(f'{response.url} bad creation', extra=response.json())
+    return response.status_code == 201
```
Comment (elreplicante) [inline postoffice_django/config.py]:
```suggestion
    return response.status_code == HTTPStatus.CREATED
```

Comment (jjponz) [reply postoffice_django/config.py]: Thanks a lot!

File: postoffice_django/exceptions.py [hunk 2]
```diff
@@ -1,13 +1,30 @@
 class UrlSettingNotDefined(Exception):
     def __init__(self):
-        self.message = "POST_OFFICE_URL is not defined on settings"
+        self.message = 'POST_OFFICE_URL is not defined on settings'
```
Comment (lonamiaec) [inline postoffice_django/exceptions.py]:
```suggestion
        self.message = 'POSTOFFICE_URL is not defined on settings'
```

File: postoffice_django/exceptions.py [hunk 3]
```diff
@@ -1,13 +1,30 @@
...
 
 
 class ConsumersSettingNotDefined(Exception):
     def __init__(self):
-        self.message = "POST_OFFICE_CONSUMERS is not defined on settings"
+        self.message = 'POST_OFFICE_CONSUMERS is not defined on settings'
```
Comment (lonamiaec) [inline postoffice_django/exceptions.py]:
```suggestion
        self.message = 'POSTOFFICE_CONSUMERS is not defined on settings'
```

File: postoffice_django/management/commands/configure_post_office_publishers.py [hunk 4]
```diff
@@ -0,0 +1,10 @@
+from django.core.management.base import BaseCommand
```
Comment (lonamiaec) [inline postoffice_django/management/commands/configure_post_office_publishers.py]: Filename should be `configure_postoffice_publishers.py`.

File: postoffice_django/management/commands/configure_post_office_topics.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+from django.core.management.base import BaseCommand
```
Comment (lonamiaec) [inline postoffice_django/management/commands/configure_post_office_topics.py]: Filename too

File: tests/test_commands.py [hunk 6]
```diff
@@ -5,9 +5,18 @@
...
+        call_command('configure_post_office_publishers')
+
+        config_mock.assert_called()
+
+
+@pytest.mark.django_db
+class TestConfigureTopicsCommand:
+    @patch('postoffice_django.config.configure_topics')
+    def test_config_command(self, config_mock):
+        call_command('configure_post_office_topics')
```
Comment (lonamiaec) [inline tests/test_commands.py]:
```suggestion
        call_command('configure_postoffice_topics')
```

File: tests/test_commands.py [hunk 7]
```diff
@@ -5,9 +5,18 @@
 
 
 @pytest.mark.django_db
-class TestConfigCommand:
-    @patch('postoffice_django.config.configure')
+class TestConfigurePublishersCommand:
+    @patch('postoffice_django.config.configure_publishers')
     def test_config_command(self, config_mock):
-        call_command('configure_post_office')
+        call_command('configure_post_office_publishers')
```
Comment (lonamiaec) [inline tests/test_commands.py]:
```suggestion
        call_command('configure_postoffice_publishers')
```

**Comment by jjponz** [comment]
I go to rename all pos_office to postoffice, thanks a lot for all the comments about this @lonamiaec  (this is quickly than go comment by comment :laughing: )

**Pull request merged**